README.md

```diff
@@ -122,6 +122,28 @@ LISK_DESKTOP_URL="http://localhost:8080" DEBUG=true npm run start
 
 This comes with Redux dev tools.
 
+### How to use ledger hardware wallet
+
+#### Step 1: Manually add the Lisk ledger app to your Ledger hardware wallet
+
+- Download `installer_nanos.sh` or `installer_nanos_plus.sh` from [ledger-lisk](https://github.com/Zondax/ledger-lisk/releases). Depending on your HW test device.
+- run `chmod +x ./installer.sh`
+- Make sure you have python3 on your environment
+- Connect your ledger nano to your computer
+- run `./installer.sh load` // to push the Lisk app to your ledger)
+- You might need to do pip install ledgerblue . Make sure you use pip for python3, depending on how you have set up your aliases to handle python2 and python3. I use pip3 for python3, in that case do pip3 install ledgerblue .
```

Comment:
```suggestion
- It may be necessary to perform a pip install ledgerblue. Ensure you use pip for python3, depending on how you have set up your aliases to handle python2 and python3. If you are using pip3 for python3, then in this case perform a pip3 install ledgerblue.
```